main.py

```diff
@@ -102,6 +102,12 @@
         methods=["PATCH"],
         status_code=status.HTTP_200_OK,
     )
+    app.add_api_route(
+        "/api/bots/{bot_id}",
+        endpoint=bot_controller.delete_chatbot,
+        methods=["DELETE"],
+        status_code=status.HTTP_204_NO_CONTENT,

```

Comment:
I have to question this practice - would it not be better to follow the existing standard in our codebase and return a 200_OK, along with a confirmation message inside a json? @jasonirvine76

Reply:
For the response status, I followed the suggestion from this blog where if the DELETE response has no content, it's better to use 204. https://stackoverflow.com/questions/2342579/http-status-code-for-update-and-delete

Considering that the code doesn't return any content and has no need to return any content, I think it's better to return this status code.

Reply:
Just realized I tagged the wrong person, my bad 😅 
That makes sense, but from a maintainability perspective I'm also thinking about how unexpected it would be to have one singular request that doesn't return any json data.

Reply:
I see, what would you suggest as the return data? Maybe the same as create and update bot where they return the detail message?

Reply:
Yes, something like that would be good in my opinion. I am aware this might be lower priority so feel free to make an _Issue_ and implement this in the final week of this sprint instead

Reply:
All right, will do it in the upcoming PR next week 👍